src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -12,74 +12,183 @@ import { useSelector } from "hooks/useSelector";
+      });
+
+      return acc;
+    }, {});
+  }, [usersInVenue]);
+
+  // TODO: do we need this?
+  // const [showEventSchedule, setShowEventSchedule] = useState(false);
 
   return (
     <>
       <div className="party-venue-container">
+        <AnnouncementMessage message={currentVenue.bannerMessage} />
+
+        {/* TODO: should this still be here on the partymap? */}
+        {/*{usersInCamp && (*/}
+        {/*  <div className="row">*/}
+        {/*    <div className="col">*/}
+        {/*      <UserList*/}
+        {/*        users={usersInCamp}*/}
+        {/*        imageSize={50}*/}
+        {/*        disableSeeAll={false}*/}
+        {/*        isCamp={true}*/}
+        {/*        activity={currentVenue.activity ?? "partying"}*/}
+        {/*      />*/}
+        {/*    </div>*/}
+        {/*  </div>*/}
+        {/*)}*/}
+
+        {/* TODO: should this still be here on the partymap? */}
+        {/*<div className="row">*/}
+        {/*  <div className="col">*/}
+        {/*    <div className="starting-indication">*/}
+        {/*      {venue.description?.text}{" "}*/}
+        {/*      {venue.description?.program_url && (*/}
+        {/*          <a*/}
+        {/*              href={venue.description.program_url}*/}
+        {/*              target="_blank"*/}
+        {/*              rel="noopener noreferrer"*/}
+        {/*          >*/}
+        {/*            Event Program here*/}
+        {/*          </a>*/}
+        {/*      )}*/}
+        {/*    </div>*/}
+        {/*    {venue.start_utc_seconds && (*/}
+        {/*        <CountDown*/}
+        {/*            startUtcSeconds={venue.start_utc_seconds}*/}
+        {/*            textBeforeCountdown="Party begins in"*/}
+        {/*        />*/}
+        {/*    )}*/}
+        {/*  </div>*/}
+        {/*</div>*/}
+
         <Map
-          venue={venue}
+          venue={currentVenue}
+          partygoers={usersInVenue}
           attendances={attendances}
           selectedRoom={selectedRoom}
-          setSelectedRoom={setSelectedRoom}
-          setIsRoomModalOpen={setIsRoomModalOpen}
+          selectRoom={selectRoom}
+          unselectRoom={unselectRoom}
         />
+
+        {/* TODO: should this still be here on the partymap? */}
+        {/*<div className="row">*/}
+        {/*  <div className="col">*/}
+        {/*    <RoomList*/}
+        {/*        rooms={venue.rooms}*/}
+        {/*        attendances={attendances}*/}
+        {/*        setSelectedRoom={setSelectedRoom}*/}
+        {/*        setIsRoomModalOpen={setIsRoomModalOpen}*/}
+        {/*    />*/}
+        {/*  </div>*/}
+        {/*</div>*/}
+
         <RoomModal
           show={isRoomModalOpen}
           room={selectedRoom}
-          onHide={modalHidden}
+          onHide={closeRoomModal}
+          // joinButtonText={currentVenue.joinButtonText} // TODO: this existed on Camp, is it just a deficiency in our types that it doesn't exist on  PartyMapVenue?
```

Comment:
We probably don't need this venue-wise, but it could be cool to be able to customize it per-room. So if we add it there.. that would work better probably.

Reply:
I believe @thecadams said this was legacy. In any case, going to mark this resolved as it no longer exists in current branch